tests/iostability/test_iostability_happy_path.py

```diff
@@ -87,3 +91,47 @@ def test_bucket_object_crud_s3bench(self):
         self.test_completed = True
         self.log.info(
             "ENDED: Test for Bucket and  Object CRUD operations in loop using S3bench for 30 days")
+
+    @pytest.mark.lc
+    @pytest.mark.io_stability
+    @pytest.mark.tags("TEST-40041")
+    def test_disk_near_full_s3bench(self):
+        """Perform disk storage near full once and read in loop for 30 days."""
+        self.log.info("STARTED: Perform disk storage near full once and read in loop for 30 days.")
+        resp = self.dfr.get_user_data_space_in_bytes(master_obj=self.master_node_list[0],
+                                                     memory_percent=10)
+        self.log.info(f"{resp[1]} bytes")
+        if resp[1] is not 0:
+            self.log.info(f"Need to add {resp[1]} for required percentage")
+            self.log.info(f"performing writes till we reach required percentage")
+            s3userinfo = dict()
+            s3userinfo['accesskey'] = ACCESS_KEY
+            s3userinfo['secretkey'] = SECRET_KEY
+            bucket_prefix = "testbkt"
+            ret = DiskFailureRecoveryLib.perform_near_full_sys_writes(s3userinfo=s3userinfo,
+                                                                      user_data_writes=int(resp[1]),
+                                                                      bucket_prefix=bucket_prefix,
+                                                                      client=20)
+            if ret[0]:
+                assert False, "Errors in write operations"
+            self.log.debug(f"{ret}")
+            end_time = datetime.now() + timedelta(days=30)
+            loop = 1
+            while datetime.now() < end_time:
+                self.log.info(f"{end_time - datetime.now()} remaining time for reading loop")
+                read_ret = DiskFailureRecoveryLib.perform_near_full_sys_operations(
+                                                                        s3userinfo=s3userinfo,
+                                                                        workload_info=ret[1],
+                                                                        skipread=False,
+                                                                        validate=True,
+                                                                        skipcleanup=True)
+                loop += 1
+                self.log.info(f"{loop} interation is done")
```

Comment:
This log should be before incrementing loop variable.

Reply:
done